### PR TITLE
Update environment.yml to use vtr-optimized (vtr compiled with PGO)

### PR DESCRIPTION
**Pull request by HackerFoo**

### CODE REVIEW

File: environment.yml
Comment (acomodi) [inline environment.yml:14]:
```suggestion
  - litex-hub::vtr-optimized=v8.0.0_3049_g64d15e2dd=20201216_065815
```

**Review by acomodi — CHANGES_REQUESTED** [review]

File: environment.yml
Comment (HackerFoo) [reply environment.yml]: Thanks.

**Comment by HackerFoo** [comment]
@mithro Could you help me figure out why Conda fails? `make env` works on my workstation.

**Comment by HackerFoo** [comment]
My theory is that the condarc file in the package causes this, because `vtr-gui` also fails: https://github.com/HackerFoo/symbiflow-arch-defs/tree/test-ci

**Comment by HackerFoo** [comment]
I think I know what's happening. Some dependencies are coming from `conda-forge`, so that needs to be included in `environment.yml`.

**Comment by acomodi** [comment]
The fact that there are some conda-forge dependencies is a hint that there is something wrong with the vtr-optmized package. Looking at the package description in litex-hub-conda, I see that there is a condarc file which should be deleted from there: https://github.com/litex-hub/litex-conda-eda/blob/master/pnr/vtr-optimized/condarc.

The problem is that conda-forge generates this kinds of issues and leads to instability, and we should not use it at all.

**Comment by HackerFoo** [comment]
Unfortunately, there is no other good way to get `python-constraints`. I can't use pip: https://github.com/conda/conda-build/issues/548

**Comment by HackerFoo** [comment]
I have created a package for `python-constraint` to avoid depending on `conda-forge`.

File: environment.yml
Comment (acomodi) [inline environment.yml:14]:
```suggestion
  - litex-hub::vtr-optimized=v8.0.0_3049_g64d15e2dd=20201217_025303
```

AFAIK, only the packages labeled as `main` are available. Using a non-main labeled package results in conda not being able to find it.

**Review by acomodi — CHANGES_REQUESTED** [review]

File: environment.yml
Comment (HackerFoo) [reply environment.yml]: Fixed. I was still waiting on the `main` package, so I thought I'd try it.

**Review by acomodi — APPROVED** [review]
LGTM, let's wait for CI

**Review by litghost — APPROVED** [review]
LGTM, merge if green

**Comment by litghost** [comment]
I'm not seeing an improvement in overall runtime.  This is likely because most of the time is spent during the lookahead computation.  @HackerFoo have you explored/tested PGO improvements on the lookahead computation stuff?

**Comment by HackerFoo** [comment]
I have not. It might be worth trying a separate PGO build optimized for lookup computation, but it's probably not worth the ~10% performance gain, and I suspect that lookup is mostly memory bound, so it will probably have less effect.

I think the most effective way to improve lookup time is by balancing the parallel workload better, and shortening the tail.

**Pull request merged**
